Фикс GregTech

SleepyG11 opened the pull request:
Исправление большей части текстур в GregTech, а так-же изменение Луны, Солнца и блока стекла (из-за визуального бага).

SleepyG11 committed:
Фикс GregTech

Исправление большей части текстур в GregTech а так-же изменение Луны, Солнца и блока стекла.

TheAndrey commented:
Слишком много файлов. Не следует включать стандартные x16 текстуры (пак должен содержать только изменённые варианты). Будущим текстурам можно временно добавлять префикс `TODO_`, чтобы они в коммит не попадали.
И нельзя трогать ванильные текстуры. Это не соотвествует общей стилистике ресурспака.

SleepyG11 commented:
К 16х16 текстурам, которые действительно нужны, относятся некоторые руды (так как есть проблема с их текстурами: как пример Graphite Ore практически невозможно отличить от обычного камня без помощи Waila) и компоненты микросхем (из-за изменений в GregTech детали микрух поменяли свой ID по metaitem, но в Faithful это не изменилось, из-за чего некоторым деталям и даже микросхемам присуща совсем не та текстура). Остальные ненужные файлы или пустые изображения я удалю, чтобы уменьшить их кол-во.
Насчёт ванилы - хорошо, тоже уберу.

TheAndrey commented:
Так в чём проблема увеличить разрешение текстур? Кубы "из коробки" теперь 32x32 поддерживают.

SleepyG11 commented:
Это не особая проблема. Я могу увеличить, если это нужно.

SleepyG11 committed:
Убраны лишние файлы и возвращены текстуры ваниллы.

Убраны все ненужные и пустые файлы, убраны микросхемы, а так-же руды (требуют долгой доработки экспериментальным методом).
